n_helpers.c

```diff
@@ -389,33 +436,33 @@ const char * NoteBinaryTransmit(uint8_t * data, size_t dataLen, size_t bufLen, s
                 // On errors, we restore the caller's input buffer by COBS
                 // decoding it. The caller is then able to retry transmission
                 // with their original pointer to this buffer.
-                cobsDecode(data, encLen, '\n', data);
+                NoteBinaryDecode(data, outLen, data, NULL);
```

Comment:
Don't love the `NULL` parameter usage here, should be `bufLen`, no?

Reply:
https://github.com/blues/note-c/pull/90#discussion_r1304978894